soperator/modules/slurm/variables.tf

```diff
@@ -7,6 +7,20 @@ variable "operator_version" {
   type = string
 }
 
+# region PartitionConfiguration
+
+variable "slurm_partition_config_type" {
+  description = "Could be default or custom"
+  default     = "default"
+  type = string
+}
+
+variable "slurm_partition_raw_config" {
+  description = "Configuration as list strings started with PartitionName"
```

Comment:
```suggestion
  description = "Type of the Slurm partition config. Could be either `default` or `custom`."
  default     = "default"
  type = string
}

variable "slurm_partition_raw_config" {
  description = "Partition config in case of `custom` slurm_partition_config_type. Each string must be started with `PartitionName`."
```